integration-tests/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/onsi/gomega v1.19.0
 	github.com/rs/zerolog v1.26.1
 	github.com/satori/go.uuid v1.2.0
-	github.com/smartcontractkit/chainlink-testing-framework v1.1.22-0.20220511162451-b8f31235228e
+	github.com/smartcontractkit/chainlink-testing-framework v1.1.23-0.20220516102704-9b6d516359af
```

Comment:
Temporarily pointing to commit in https://github.com/smartcontractkit/chainlink-testing-framework/pull/304 . Will update to exact version once it's cut